sql/table_trigger_dispatcher.h

```diff
@@ -63,7 +68,9 @@ class Table_trigger_dispatcher : public Table_trigger_field_support {
  public:
   static Table_trigger_dispatcher *create(TABLE *subject_table);
 
-  bool check_n_load(THD *thd, const dd::Table &table);
+  bool initial_load(THD *thd, const dd::Table &table);
+
+  bool finalize_load(THD *thd);
```

Comment:
Not easily. In theory, we can acquire dd::Table object from DD API and pass it to finalize_load(). But this might be a bit expensive scalability and code complexity-wise. Let me see if we can store necessary information associated with TABLE_SHARE, load it once during TABLE_SHARE construction and do not have the same information duplicated for each TABLE object.

Reply:
Follow-up patch solves this problem by doing what was described above.